AllReadyApp/Web-App/AllReady/Views/Activity/Activity.cshtml

```diff
@@ -223,30 +223,18 @@
                                         <span asp-validation-for="SignupModel.AdditionalInfo" class="text-danger"></span>
                                     </div>
                                 </div>
-                                @if (Model.RequiredSkills?.Count > 0)
-                                {
-                                    <div class="form-group">
+                                    <div class="form-group" data-bind="visible: skills.filter(function(skill){ return !skill.IsUserSkill; }).length > 0">

```

Comment:
For cleaner code and easier debugging, make `skills.filter(function(skill){ return !skill.IsUserSkill; })` part of the view model (it does not need to be an observable):

In the `ActivityViewModel` constructor of `activity.js`:
```JavaScript
this.skills = skills.map(RequiredSkill.bind(null, userSkills));
this.unassociatedSkills = this.skills.filter(function(skill){ return !skill.IsUserSkill; }); //added
```